frontend/src/pages/GraphPF/styles/styleEdge.tsx

```diff
@@ -31,7 +124,7 @@ const StyleEdgeComponent: React.FC<StyleEdgeProps> = ({ element, ...rest }) => {
     return newData;
   }, [data, detailsLevel]);
 
-  return <BaseEdge element={element} tagClass={tagClass} {...rest} {...passedData} />;
+  return <DefaultEdge className={classes?.join(' ')} element={element} {...rest} {...passedData} />;
```

Comment:
Instead of `join(' ')` I would use `classes` method of `typestyle` library, a utility to join CSS classes conditionally
```
<DefaultEdge className={classes(...cssClasses)} element={element} {...rest} {...passedData} />;
```